server/routes/student/index.ts

```diff
@@ -0,0 +1,5 @@
+import { Router } from 'express';
+
+const studentRouter = Router();
```

Comment:
You can do it like this especially that you are exporting it by default
```suggestion
const router = Router();
```